bundles/org.openhab.binding.allplay/pom.xml

```diff
@@ -13,13 +13,10 @@
 
   <name>openHAB Add-ons :: Bundles :: AllPlay Binding</name>
 
-  <dependencies>
```

Comment:
Will it still compile in the IDE if the dependency is not declared anymore (and embedded only at build time, if I see it correctly)?

Reply:
Didn‘t check. But I assume we can keep the dependency since we provide all necessary packages it should work even if the bundle is not explicitly added to the feature.

Reply:
I also added lifecycle mapping to m2e, otherwise we would get the red X on the pom.xml.